### PR TITLE
Vendor location, & Event svc

### PR DESCRIPTION
**Pull request by depakaakula**
This reverts commit e1225f2ac609f0a94f7ac33f84f2d44435baaf46.

### CODE REVIEW

File: food-truck/src/app/pages/create-event/create-event.component.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 import { FormBuilder, FormGroup, FormArray } from '@angular/forms';
+import { MapsAPILoader } from '@agm/core';
+import {} from '@types/googlemaps';
```
Comment (szwo) [inline food-truck/src/app/pages/create-event/create-event.component.ts]: I think the right syntax here is `import` * from  '@types/googlemaps`

File: food-truck/src/app/pages/create-event/create-event.component.css [hunk 2]
```diff
@@ -1,3 +1,53 @@
+#map {
+        height: 100%;
+      }
+      /* Optional: Makes the sample page fill the window. */
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
```
Comment (szwo) [inline food-truck/src/app/pages/create-event/create-event.component.css]: Probably shouldn't have tags in here like this, right?

File: food-truck/src/app/pages/create-event/create-event.component.ts (hunk 1 above)
Comment (depakaakula) [reply food-truck/src/app/pages/create-event/create-event.component.ts]: this was recommended for agm and stackoverflow for google maps api.

File: food-truck/src/app/pages/create-event/create-event.component.css (hunk 2 above)
Comment (depakaakula) [reply food-truck/src/app/pages/create-event/create-event.component.css]: corrected

**Pull request merged**